test/statistic_test.cc

```diff
@@ -269,6 +272,19 @@ TEST(StatisticTest, StreamingStatProtoOutputLargeValues) {
   EXPECT_EQ(proto.pstdev().nanos(), 0);
 }
 
+TEST(StatisticTest, CircllhistStatisticProtoOutputLargeValues) {
+  CircllhistStatistic a;
+  uint64_t value = 100ul + 0xFFFFFFFF;
```

Comment:
I would find a named const instead of `0xFFFFFFFF` slightly easier to read, but that is probably personal, so feel free to ignore :-)